### PR TITLE
adding release notes for version 12

### PR DESCRIPTION
**Pull request by borgmanJeremy**

### CODE REVIEW

File: docs/ReleaseNotes_12.0.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+- Added magnifier for more precise selections by @SilasDo in https://github.com/flameshot-org/flameshot/pull/2219
+  - The new magnifier can be enabled in ```Configuration > General > Show Magnifier``` 
+  - There is an option to make the magnifier a square or circle
+<p align=center><img src="https://github.com/flameshot-org/flameshot/blob/master/docs/images/magnfier.png" width=50%> </p>
+
+- Incremental markers can now have a point if you drag when placing them. @vozdeckyl in https://github.com/flameshot-org/flameshot/pull/2638
+<p align=center><img src="https://github.com/flameshot-org/flameshot/blob/master/docs/images/number_pointer.png" width=50%> </p>
+
+- Added the ability to cache the last region by @borgmanJeremy in https://github.com/flameshot-org/flameshot/pull/2615
+  - The launcher tool will automatically populate the coordinates for the last selection region
```
Comment (mmahmoudian) [inline docs/ReleaseNotes_12.0.md:21]: I suggest adding a screenshot for this.

For example:

image

File: docs/ReleaseNotes_12.0.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+- Added 'Save to disk' button when uploading to imgur by @AndreaMarangoni in https://github.com/flameshot-org/flameshot/pull/2237
+
+- Pinned screenshots can now be zoomed with a pinch gesture by @AndreaMarangoni in https://github.com/flameshot-org/flameshot/pull/2447
+
+- The SVG's have been optimized by @RiedleroD in https://github.com/flameshot-org/flameshot/pull/2318
+
+- Make KDE use Freedesktop portal by @greenfoo in https://github.com/flameshot-org/flameshot/pull/2495
+
+- Allow final actions when printing geometry by @borgmanJeremy in https://github.com/flameshot-org/flameshot/pull/2444
```
Comment (mmahmoudian) [inline docs/ReleaseNotes_12.0.md:36]: Can this be a good addition?

> Allow final actions when printing geometry <ins>when invoked by CLI</ins>

File: docs/ReleaseNotes_12.0.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+- Many Flameshot widgets have been reworked to use .ui XML files and Qt Designer. This has been done to allow non C++ developers to more easily contribute to the graphical side of Flameshot.
+<p align=center><img src="https://github.com/flameshot-org/flameshot/blob/master/docs/images/ui_file.png" width=50%> </p>
+
+- Updated Translations
+
```
Comment (mmahmoudian) [inline docs/ReleaseNotes_12.0.md:42]: Was it in this version that right-clicking the tools would bring up its settings in the sidebar?

**Review by mmahmoudian — COMMENTED** [review]
Very nice and detailed. Thanks! :+1:

File: docs/ReleaseNotes_12.0.md (hunk 3 above)
Comment (borgmanJeremy) [reply docs/ReleaseNotes_12.0.md:42]: Nope that seems to always work on v11 (I had to test to check lol)

**Pull request merged**